### ISSUE TITLE
Make "Play" fail gracefully if spotify has a pending "enable-drm" prompt (currently fails with "Internal error: Error: Could not find element a[aria-label='Search']")

### ISSUE THREAD

**Issue by dholbert** [mozilla/firefox-voice#551]
STR:
 1. Start with a fresh Firefox profile.
 2. Visit https://open.spotify.com/browse and sign in.
 --> A yellow "You must enable DRM" dropdown should appear.  **Leave it there -- do not click through it.**
 3. Install Firefox Voice, and click its toolbar icon, and say or type "play ben folds in spotify"

EXPECTED RESULTS:
Firefox Voice should give me a human-readable message saying something like "Sorry, we can't find the search field in your Spotify tab" or something like that.

ACTUAL RESULTS:
Tab switches to spotify, and Firefox Voice popup says: "Sorry, there was an issue" / "Internal error: Error: Could not find element a[aria-label='Search']"
![image](https://user-images.githubusercontent.com/426803/68499613-755c4380-020e-11ea-8827-a2ced65ad9ab.png)

(Basically, Firefox Voice isn't prepared for how Spotify looks when it's in a you-still-need-to-enable-DRM state, and it fails awkwardly rather than gracefully right now.)

See also previously-fixed-bugs #441 and #454 (which seem to be about similar failures for other types of unexpected Spotify states).

I'm using Firefox Voice version 0.12.0 in latest Firefox Nightly 72.0a1 (2019-11-08)

**Comment by alexandra-martin** [comment]
Managed to reproduce the bug following the steps mentioned above with some additional changes. 

With a new Firefox Nightly profile, I was able to install Firefox Voice either before or after accessing and signing in to the Spotify website, as the "Play DRM-controlled content" option from the "Digital Rights Management (DRM) Content" section available in `about:preferences` needed to be manually disabled, in order for the yellow "You must enable DRM" dropdown to appear on the Spotify website.
After clicking on the mic icon and making the "Play Ben Folds in Spotify" or "Play Green Day" commands the "Internal error: Error: Could not find element a[aria-label='Search']" error appeared in the "Doorhanger".
Adding the bug label.

Reproduced on Mac 10.14.6 and Win10x64 with Firefox Nightly 72.0a1 (64-bit).

![5](https://user-images.githubusercontent.com/54401091/68586735-f9702e80-048d-11ea-8074-390247b8c596.gif)

**Comment by ianb** [comment]
We should scan for uses of "on" and also support "in"

**Comment by ianb** [comment]
The fix for this would be to detect the error message, and construct an exception with `.displayMessage = "You must enable DRM"` or something like that.

**Comment by MLCoco** [comment]
> The fix for this would be to detect the error message, and construct an exception with `.displayMessage = "You must enable DRM"` or something like that.

I'm working on this bug right now and I'm wondering what would be a good way to detect the error message?
Right now, the DOM contains an element that links to Spotify's DRM support page, so I can select that.

**Comment by alexandra-martin** [comment]
Verified on Mac 10.14 and Win 10x64 with Firefox Nightly 76.0a1 (2020-03-16).

![1](https://user-images.githubusercontent.com/54401091/76764039-db4c6f00-679c-11ea-80b0-4f7a89efc97c.gif)
